data/eos-companion-app.service

```diff
@@ -6,7 +6,30 @@ ConditionPathExists=/etc/avahi/services/companion-app.service
 [Service]
 ExecStart=/usr/bin/dbus-run-session /usr/bin/flatpak run --no-desktop --no-a11y-bus --no-documents-portal com.endlessm.CompanionAppService
 User=companion-app-helper
+# This does not appear to cause any problems when writing
+# to /var/lib/eos-companion-app, remove if it does. We need
+# CAP_SETUID and CAP_SETGID as bwrap is setuid. CAP_SYS_ADMIN,
+# CAP_NET_ADMIN and CAP_SYS_CHROOT are also used by bwrap.
+CapabilityBoundingSet=CAP_NET_ADMIN CAP_SYS_ADMIN CAP_SYS_CHROOT CAP_SETUID CAP_SETGID
+MemoryDenyWriteExecute=yes
+NoNewPrivileges=yes
+PrivateDevices=yes
+# Breaks xapian-bridge, since it needs local networking
```

Comment:
I think the wording here is unclear: “Breaks xapian-bridge” implies that `PrivateNetwork=no` breaks xapian-bridge, rather than `PrivateNetwork=yes` breaks it. I suggest rewording to “xapian-bridge needs local networking”.

Same for the other comment below.